README.md

```diff
@@ -86,6 +86,19 @@ Lastly, validate if the content above the accessory view has correct offsets, if
 
 - `spaceBetweenKeyboardAndAccessoryView` (optional) - accepts a number. Use to adjust space between the accessory view and the keyboard, when the latter is open. Read more [here](#handling-wrong-offsets).
 
+- `renderBackgroundElement` (optional) - you can provide a render function to attach an extra node to the wrapper. This is useful when you want to have a custom element as the background (eg. `BlurView`):
```

Comment:
Move before `spaceBetweenKeyboardAndAccessoryView`.

To keep the style I would write this like that (also I used ImageBackground as an example, since it is a part of React Native):
```
- `renderBackgroundNode` (optional) - accepts a function returning React node. This is useful when you want to have a custom node as a background (e.g. `<ImageBackground style={StyleSheet.absoluteFill} />` ). Remember about absolute positioning.
```